schemas/descriptors/deprecated/descriptorDeprecated.example.2.json

```diff
@@ -0,0 +1,6 @@
+{
+  "@type": "xdm:descriptorDeprecated",
+  "xdm:sourceSchema": "https://ns.adobe.com/xdm/context/person-name",
+  "xdm:sourceVersion": 1,
+  "xdm:sourceProperty": ["/courtesyTitle", "suffix"]
```

Comment:
Please add a "/" to the "suffix". It will also help tp add one more example file with deprecation of nested attributes too.

Reply:
Done